client/app/components/packages/filed-eas/edit.js

```diff
@@ -5,6 +5,18 @@ import SubmittablePackageFormValidations from '../../../validations/submittable-
 import SubmittableCeqrInvoiceQuestionnaireFormValidations from '../../../validations/submittable-ceqr-invoice-questionnaire-form';
 
 export default class PackagesFiledEasEditComponent extends Component {
+  @service
+  store;
+
+  constructor(...args) {
+    super(...args);
+
+    if (this.args.package.ceqrInvoiceQuestionnaires.length < 1) {
```

Comment:
Okay interesting -- ceqrInvoiceQuestionnaires is a hasMany relationship because it's store as an array of objects in the database. BUT the business logic actually dictates that there will only ever be ONE ceqrInvoiceQuestionnaire (although a user can add more if they like, but that's not supposed to happen -- this is a CRM problem). So maybe checking for something like if `ceqrInvoiceQuestionnaires.firstObject.id` exists would be better?